lib/redisearch.rb

```diff
@@ -18,7 +18,7 @@ class RediSearch
     create: [:nooffsets, :nofreqs, :nohl, :nofields],
     del: [:dd],
     drop: [:keepdocs],
-    search: [:nocontent, :verbatim, :nostopwords, :withscores, :withsortkeys],
+    search: [:nocontent, :verbatim, :nostopwords, :withscores, :withsortkeys, :highlight, :summarize],
```

Comment:
this is great addition! 🙇 

`hightlight` and `summarize` are a little bit more powerful tho; as you can for example limit highlighted fields or declare highlight tag (see: https://oss.redislabs.com/redisearch/Commands.html#ftsearch)

I think moving this to `OPTIONS_PARAMS` should _mostly_ solve it 🤔 

@vruizext are you still planning to maintain this gem?